lib/src/commands/chat_command.dart

```diff
@@ -432,6 +458,8 @@ class ChatCommand
     Iterable<AbstractCheck> checks = const [],
     Iterable<AbstractCheck> singleChecks = const [],
     this.options = const CommandOptions(),
+    this.localizedNames,
+    this.localizedDescriptions,
   }) {
     if (!commandNameRegexp.hasMatch(name) || name != name.toLowerCase()) {
```

Comment:
Maybe we should check localized name validity here too - also applies to User and Message Commands

Reply:
Ah yes, just for chat commands then.